packaging/systemd/microshift-containerized-configure-selinux

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+sudo dnf -y install container-selinux
+sudo dnf -y install selinux-policy-devel 
+curl -L -o /tmp/microshift.fc https://raw.githubusercontent.com/redhat-et/microshift/main/packaging/selinux/microshift.fc
+curl -L -o /tmp/microshift.te https://raw.githubusercontent.com/redhat-et/microshift/main/packaging/selinux/microshift.te
+make -f /usr/share/selinux/devel/Makefile -C /tmp
+sudo dnf -y remove selinux-policy-devel
```

Comment:
```suggestion
```

If we keep this script, please let's stop removing the devel package :-)

install.sh removes it on my system, which I never expect and it's annoying.